tests/setup.js

```diff
@@ -4,6 +4,10 @@
  */
 // eslint-disable-next-line node/no-extraneous-import
 import 'regenerator-runtime/runtime'
+
+if (typeof structuredClone === 'undefined') {
+	global.structuredClone = (val) => JSON.parse(JSON.stringify(val))
+}
```

Comment:
Available since Node 17 so should not be needed?

Reply:
Tests fail without it for some reason